CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contribution Guide
+
+## Releasing a new version
+
+### Building the new version
+
+Increment the version in `pyproject.toml`.
+
+Install development dependencies:
+```bash
+pip install ".[dev]"
+```
+
+Build a source and a wheel distributions:
+```bash
+rm -rf ./dist # clean the build directory if necessary
+python -m build
+```
+
+### Publishing to the Test Pypi
+
+Login to the Test PyPi. Create a new account if you don't have it yet
+and ask to be added as a collaborator for Anonymeter.
+
+Get the token from the Test PyPi: https://test.pypi.org/manage/account/#api-tokens
+and save it as suggested to `$HOME/.pypirc`:
+```toml
+[testpypi]
+  username = __token__
+  password = YOUR_TOKEN_HERE
+```
+
+Upload the artifacts to the Test PyPi:
+```bash
+twine upload --repository testpypi dist/*
+```
+
+Test the package can be installed and works properly. For example,
+you can create a new virtualenv and try to install the package there.
+```bash
+mkdir ~/test-anonymeter # create some test directory
+cd ~/ test-anonymeter
+python -m venv .venv # create new virtual env
+source .venv/bin/activate
+asdf reshim python # in case you use asdf
+pip install --upgrade pip
+pip install --index-url https://test.pypi.org/simple anonymeter==NEW_VERSION
+```
+
+You can check that anonymeter is working by running it against the original tests:
+```
+ln -s ~/code/anonymeter/tests ./tests
```

Comment:
It's not clear what the source/destination of the soft link are. Should we use complete paths to remove the ambiguity? E.g.:

```suggestion
ln -s ~/code/anonymeter/tests ~/test-anonymeter/tests
```

(if I understood your intention correctly)